addons/hr_holidays/security/hr_holidays_security.xml

```diff
@@ -38,7 +38,7 @@
     <record id="hr_leave_rule_officer" model="ir.rule">
         <field name="name">Leaves Officer</field>
         <field name="model_id" ref="model_hr_leave"/>
-        <field name="domain_force">['|','|','|', ('employee_id.user_id', '=', user.id), ('department_id', '=', False), ('department_id.manager_id', '=', False), ('department_id.manager_id.user_id', '=', user.id)]</field>
+        <field name="domain_force">[('employee_id.company_id', 'child_of', user.company_ids.ids)]</field>
```

Comment:
doesn't child of requires the main company only? it seems weird to display all company you have access to instead of the company you're currently logged into (i.e. when in Odoo S.A., I only expect to see Odoo S.A. holidays since my `company_id` is 1, even though my `company_ids` is [1,2,3,4,5])